client/src/component/admin/__test__/MenuForm.test.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import userEvent from "@testing-library/user-event";
+
+import MenuForm from "../MenuForm";
+
+describe("should render form and form element", () => {
+  it("should render item name input", () => {
+    render(<MenuForm values={{ itemname: "", price: "", categoryId: "" }} />);
+
+    const inputNode = screen.getByTestId("item-name");
+    expect(inputNode).toBeInTheDocument();
+  });
+  it("should render item price input", () => {
+    render(<MenuForm values={{ itemname: "", price: "", categoryId: "" }} />);
+
+    const inputNode = screen.getByTestId("price");
+    expect(inputNode).toBeInTheDocument();
+  });
+
+  it("should correctly set default option", () => {
+    render(<MenuForm values={{ itemname: "", price: "", categoryId: "" }} />);
+    expect(screen.getByRole("option", { name: "Select option" }).selected).toBe(
+      true
+    );
+  });
+  it("should display the correct number of options", () => {
+    render(<MenuForm values={{ itemname: "", price: "", categoryId: "" }} />);
+
+    expect(screen.getAllByRole("option").length).toBe(5);
+  });
+});
+
+describe("userEvent", () => {
+  it("should item name in the document", () => {
+    render(<MenuForm values={{ itemname: "", price: "", categoryId: "" }} />);
+    const inputNode = screen.getByTestId("item-name");
+    userEvent.type(inputNode, "Coffee");
+    expect(inputNode).toBeInTheDocument("Coffee");
```

Comment:
I think you mean:
 
```suggestion
    expect(inputNode).toHaveValue("Coffee");
```

`toBeInTheDocument` doesn't accept an argument.

Reply:
![Screenshot 2022-11-17 at 6 29 27 PM](https://user-images.githubusercontent.com/56571107/202603073-74239aad-03a4-49bd-b175-352964c1bc12.png)